tools/ete/ete_lineage_generator.xml

```diff
@@ -68,17 +69,16 @@ python '$__tool_directory__/ete_lineage_generator.py'
         </conditional>
         <conditional name="compresscond">
             <param name="compress" type="select" label="Fill unnamed ranks" help="Fill unnamed ranks with super/sub ranks (see -l)">
-                <option value="-c">Yes</option>
+                <option value="-c" selected="True">Yes</option>
```

Comment:
Not an issue, but that's not needed. The first option is the one selected by default.

Reply:
OK